lib/tasks/report.rake

```diff
@@ -60,6 +60,117 @@ namespace :report do
     puts
   end
 
+  desc "generate a report of counts of published PDF documents per organisation to help the Content Operating Model team"
+  task pdf_content_operating_model: :environment do
+    def unique_owning_organisation_ids
+      @_owning_organisation_ids ||= all_document_classes.map { |document| document.organisations.first }.uniq
+    end
+
+    def all_document_classes
+      @_all_document_classes ||= FinderSchema.schema_names.map do |schema_name|
+        schema_name.singularize.camelize.constantize
+      end
+    end
+
+    def organisation_name_from_id(org_id)
+      Services.publishing_api.get_content(org_id)['title']
+    end
+
+    def all_document_ids_and_version_hashes_for_document_class(document_class)
+      params = {
+        publishing_app: "specialist-publisher",
+        document_type: document_class.document_type,
+        fields: [
+          :content_id,
+          :state_history,
+        ],
+        per_page: 100000,
```

Comment:
I don't know if it'll help the timeout issues you were seeing, but I reckon we could reuse the Paginator in the `content_operating_model` report below to avoid doing this (and reduce load on publishing-api trying to return all 30,000 DFID documents in one go).